README.md

```diff
@@ -2,25 +2,16 @@
 
 :tada: Welcome to the OpenZaak market consultation! :tada:
 
-:tada: Welkom bij de OpenZaak marktconsultatie! :tada:
+This is the archive of a market consultation where market parties were invited to help lay the foundation for a community around OpenZaak. This included co-creating codebase governance, community dynamics, and the product marketing around OpenZaak with the purpose of enabling market parties to pursue a sustainable business model.
 
-## Nederlands
+Working from some assumptions that were co-created by the participating municipalities and Dimpact a set of questions were published. Eleven responses were submitted, and some of them were also published publicly in this archive. From these responses a number of workshops were prepared and executed. Thanks to the workshops some conclusions could be drawn. Below you can find all material that was created during this process.
```

Comment:
Suggest active sentence: 

"A set of questions were developed based on assumptions that were co-created by Dimpact and the participating municipalities."